test/e2e/nodepool_machineconfig_test.go

```diff
@@ -163,13 +163,35 @@ func testNodepoolMachineconfigGetsRolledout(parentCtx context.Context, mgmtClien
 			t.Fatalf("failed to update nodepool %s after adding machineconfig: %v", nodePool.Name, err)
 		}
 
+		// DS Customization
```

Comment:
I thought this PR is for n NTO, how is this change related?

Reply:
Just to follow the same pattern in every similar test. I can come back to the older one, but this looks better if you will add more tests like this in the future.

Reply:
Also, the way it is now, conflicts with the current approach in NTO, this is why I'm "fixing" the MC tests